.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "plugins": ["transform-es2015-modules-commonjs"]
+}
```

Comment:
Can we add a newline here?

<img width="227" alt="screen shot 2018-05-18 at 10 00 16 am" src="https://user-images.githubusercontent.com/67283/40242096-478ed178-5a82-11e8-82e3-63c00b784112.png">

(and i'm pretty inconsistent about this myself... i should configure my editor to force this)